ksql-common/src/main/java/io/confluent/ksql/schema/ksql/FormatOptions.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.schema.ksql;
+
+import java.util.Objects;
+import java.util.function.Function;
+
+public final class FormatOptions {
```

Comment:
With the old Connect Schema we needed to use the visitor pattern to build a `SqlSchemaFormater` class to convert the Connect schema into an SQL string, because we didn't control the schema code.

With the new `SqlType`s we control the code, so the `toString` can output SQL!

However, `SqlSchemaFormater` has a few options to control the generated SQL text, e.g. escaping reserved words, optionally adding `NOT NULL` onto non-optional fields, etc.  There is no way to do this with `toString()`. So, like others, I've added an overload of `toString` which takes a `FormatOptions` instance to control how the SQL is generated.